starter-code/scripts/views/repoView.js

```diff
@@ -2,19 +2,21 @@
 
 (function(module) {
   var repoView = {};
-  /* TODO: Let's compile our new template!
+  /* TODO: DONE: Let's compile our new template!
        Save the result of invoking Handlebars in this 'repoCompiler' variable
        that we will pass to the append method below. */
-  var repoCompiler;  // Finish the Handlebars method here!
+  var repoCompiler = Handlebars.compile($('#repo-template').html());  // Finish the Handlebars method here!
 
   repoView.renderRepos = function() {
     $('#about ul').empty().append(
-      reposObj.withTheAttribute('name')  // TODO: experiment changing this attribute!
+      reposObj.withTheAttribute('name')  // TODO: ?? experiment changing this attribute!
       .map(repoCompiler)
     );
   };
-/* TODO: Call the function that loads (or 'requests') our repo data.
+/* TODO: DONE: Call the function that loads (or 'requests') our repo data.
     Pass in some view function as a higher order callback, so our repos
     will render after the data is loaded. */
-
-});
+  reposObj.requestRepos(repoView.renderRepos);
```

Comment:
Nice job rendering the repos to the page as a callback after receiving a response from the Github API!